src/lib.rs

```diff
@@ -1665,10 +1671,33 @@ impl GooseAttack {
                 }
 
                 // Flush request statistics collected prior to all user threads running
-                if self.configuration.reset_stats && !statistics_reset {
-                    info!("statistics reset...");
-                    self.stats.requests = HashMap::new();
-                    statistics_reset = true;
+                if !users_launched {
+                    users_launched = true;
+                    if !self.configuration.no_reset_stats {
+                        self.stats.duration = self.started.unwrap().elapsed().as_secs() as usize;
+                        self.stats.print_running();
+
+                        if self.stats.users < self.users {
+                            println!(
+                                "{} of {} users hatched, timer expired, resetting statistics (disable with —no-reset-stats).\n", self.stats.users, self.users
```

Comment:
-> `--no-reset-stats`

Reply:
It appears I cut & paste this from https://github.com/tag1consulting/goose/issues/106#issuecomment-660451940 and missed that the `--` had been converted to `—`. Thanks!